meson.build

```diff
@@ -585,6 +605,16 @@ subdir('src/lxc/tools/include')
 subdir('src/tests')
 subdir('templates')
 
+# Pkg-config.
+pkg_config_file = pkgconfig.generate(liblxc,
+    description: 'linux container tools',
+    version: version_data.get('LXC_VERSION'),
+    url: 'http://linuxcontainers.org',
+    libraries: '-lutil -lpthread -ldl',
+    libraries_private: pkgconfig_libs,
```

Comment:
If you are not confident that Meson's autodetection will get this correct, you should still pass them as the returned dep objects from `dependency()` so that Meson can properly set these up via pkg-config instead of open-coded link flags.

Reply:
I just tried that and passing `depdendency('name')` doesn't work. It doesn't fail but leads to an empty list.
Using `cc.find_library` does work, but at that point my current approach is simpler.

Reply:
Oh, I see, it's moving it to Requires.private instead and uses pkgconfig names instead, I guess that should be fine.